app/js/components/names.jsx

```diff
@@ -134,7 +134,12 @@ class Name extends React.Component {
     }
     return (
       <div>
-        <form className='form-horizontal'>
+          <h3 className=''
+            name=''
+          > {this.state.givenName} {' '} {this.state.middleName } {''} {this.state.familyName}</h3>
+          <a> Edit </a>
+
```

Comment:
You have classname and name properties for the <h3> tag but not assigned to anything, you could just omit them instead if they serve no purpose.